debian/rules

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/make -f
+
+export DH_VERBOSE = 1
+VERSION ?= $(shell cat VERSION)
+TARGET_ARCH = $(shell dpkg-architecture -qDEB_TARGET_ARCH)
+# export AUTO_GOPATH=1
+export GOPROXY=direct
+export GO111MODULE=off
+export GOPATH=/go
+export GOROOT=/usr/local/go
+# export PATH=$PATH:/usr/local/go/bin
+
+# force packages to be built with xz compression, as Ubuntu 21.10 and up use
+# zstd compression, which is non-standard, and breaks 'dpkg-sig --verify'
+override_dh_builddeb:
+	dh_builddeb -- -Zxz
+
+override_dh_auto_clean:
+
+override_dh_auto_build:
+	# install go 1.16
+	curl https://dl.google.com/go/go1.16.4.linux-amd64.tar.gz -o /tmp/go.tar.gz
```

Comment:
Why are we installing golang here and not defining it in build depends ?

Reply:
bullseye's apt mirror comes with [1.15](https://tracker.debian.org/pkg/golang-defaults) but we need at least 1.16 for docker during the build process (on their side, they're building it inside container which already has this version)